event_model/test_em.py

```diff
@@ -313,3 +349,8 @@ def __call__(self, c, d):
         filler('stop', stop_doc)
         assert not filler._closed
     assert event['data']['image'].shape == (5, 5)
+
+    # Test verify_filled.
+    with pytest.raises(event_model.UnfilledData):
+        event_model.verify_filled(event_model.pack_event_page(raw_event))
+    event_model.verify_filled(event_model.pack_event_page(event))
```

Comment:
Why do we need it twice here? Or should it raise within the `pytest.raises` context manager?

Reply:
Oh, my bad. I see it's a different event (`raw_event` vs. `event`). Nevermind, looks good!